concepts/variadic-functions/about.md

```diff
@@ -0,0 +1,57 @@
+# About
+Functions in general accept only a fixed number of arguments. A variadic function is a function that accepts a variable number of arguments. If the last parameter of a function definition is prefixed by ellipsis ..., then the function can accept any number of arguments for that parameter.
+
+```go
+func hello(a int, b ...int) {
```

Comment:
Maybe it makes sense to use `find` here as well. `hello` with these int params seems a bit strange. Sometimes these example that don't make sense make it harder for students to understand the content.

Reply:
will make changes to it

Reply:
resolved